Makefile

```diff
@@ -143,13 +143,22 @@ src/ocamlbuildlib.cmxa: src/ocamlbuild_pack.cmx $(EXTRA_CMX)
 
 # The packs
 
+# Build artifacts are first placed into tmp/ to avoid a race condition
+# described in https://caml.inria.fr/mantis/view.php?id=4991.
```

Comment:
Would it not be simpler to just have `src/ocamlbuild_pack.cmx` depend on `src/ocamlbuild_pack.cmi`? This corresponds to serializing (byte before native), but it makes for simpler code (and makes sure there are no side-effects related to running in a different directory).